Automattic-Tracks-iOSTests/CrashLoggingTests.swift

```diff
@@ -0,0 +1,141 @@
+import XCTest
+@testable import AutomatticTracks
+
+class CrashLoggingTests: XCTestCase {
+
+    private let crashLogging = CrashLogging()
```

Comment:
It is indeed! After talking to @koke, we'll want to avoid using the global Sentry singleton inside of this class. I started down that road, realized it could be done later without changing the public API, and figured it'd be good to leave for later :)

Reply:
Fixed in a767cf46e0b2c19299e77fce69296774dfe18964.